src/lib.rs

```diff
@@ -389,23 +389,123 @@ impl Navigator {
         self.led.all_off()
     }
 
+    /// Reads the magnetometer Ak09915 of [`Navigator`].
+    ///
+    /// Measurements in [µT]
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use navigator_rs::{Navigator};
+    /// use std::thread::sleep;
+    /// use std::time::Duration;
+    ///
+    /// let mut nav = Navigator::new();
+    /// nav.init();
+    ///
+    /// loop {
+    ///     let read = nav.read_mag();
+    ///     println!("mag values: X={},Y={},Z={}", read.x, read.y, read.z);
+    ///     sleep(Duration::from_millis(1000));
+    /// }
+    /// ```
     pub fn read_mag(&mut self) -> AxisData {
         let (x, y, z) = self.mag.read().unwrap();
         AxisData { x, y, z }
     }
 
+    /// Reads the temperature using BMP280 of [`Navigator`].
+    ///
+    /// Measurements in [˚C]
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use navigator_rs::{Navigator};
+    /// use std::thread::sleep;
+    /// use std::time::Duration;
+    ///
+    /// let mut nav = Navigator::new();
+    /// nav.init();
+    ///
+    /// loop {
+    ///     println!("temperature value: {}", nav.read_temperature());
+    ///     sleep(Duration::from_millis(1000));
+    /// }
+    /// ```
     pub fn read_temperature(&mut self) -> f32 {
         self.bmp.temperature_celsius().unwrap()
     }
 
+    /// Reads the altitude based on pressure values measured by BMP280 of [`Navigator`].
+    ///
+    /// Measurements in [m]
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use navigator_rs::{Navigator};
+    /// use std::thread::sleep;
+    /// use std::time::Duration;
+    ///
+    /// let mut nav = Navigator::new();
+    /// nav.init();
+    ///
+    /// loop {
+    ///     println!("altitude value: {}", nav.read_altitude());
+    ///     sleep(Duration::from_millis(1000));
+    /// }
+    /// ```
     pub fn read_altitude(&mut self) -> f32 {
         self.bmp.altitude_m().unwrap()
     }
 
+    /// Reads the pressure based on BMP280 of [`Navigator`].
+    ///
+    /// Measurements in [kPa]
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use navigator_rs::{Navigator};
+    /// use std::thread::sleep;
+    /// use std::time::Duration;
+    ///
+    /// let mut nav = Navigator::new();
+    /// nav.init();
+    ///
+    /// loop {
+    ///     println!("pressure value: {}", nav.read_pressure());
+    ///     sleep(Duration::from_millis(1000));
+    /// }
+    /// ```
     pub fn read_pressure(&mut self) -> f32 {
         self.bmp.pressure_kpa().unwrap()
     }
 
+    /// Reads the ADC based on ADS1115 of [`Navigator`].
+    ///
+    /// Measurements in [bits]
```

Comment:
This measurement unit is not bits. It's 'bits' when we integrate an internet speed. Here it's the percentage of the voltage in the pin encoded/mapped in an integer from 0 to some value (possibly `2^(b)-1`, being `b` the precision in bits).

Reply:
Fixed, it will display in Volts. 
[Adc_reading](https://github.com/bluerobotics/navigator-rs/pull/17)

Reply:
/// Reads the ADC based on ADS1115 of [`Navigator`].
    ///
    /// Measurements in \[V\]
    ///
    /// Same as [`read_adc`](struct.Navigator.html#method.read_adc), but it returns an array with all channel readings